invenio_rdm_records/records/mappings/os-v2/rdmrecords/records/record-v7.0.0.json

```diff
@@ -1595,6 +1587,28 @@
             "type": "boolean"
           }
         }
+      },
+      "internal_notes": {
```

Comment:
is it outside the `metadata` object in the end? it is a bit hard to see if from the diff here

Reply:
yes, it is outside